service/src/main/kotlin/fi/espoo/evaka/Audit.kt

```diff
@@ -100,11 +100,11 @@ enum class Audit(
     DecisionReject("evaka.decision.reject"),
     DecisionsReportRead("evaka.decisions-report.read"),
     DuplicatePeopleReportRead("evaka.duplicate-people-report.read"),
-    EmployeeCreate("evaka.employee.create"),
-    EmployeeDelete("evaka.employee.delete"),
+    EmployeeCreate("evaka.employee.create", securityEvent = true, securityLevel = "high"),
+    EmployeeDelete("evaka.employee.delete", securityEvent = true, securityLevel = "high"),
     EmployeeGetOrCreate("evaka.employee.get-or-create", securityEvent = true, securityLevel = "high"),
     EmployeeRead("evaka.employee.read"),
-    EmployeeUpdate("evaka.employee.update"),
+    EmployeeUpdate("evaka.employee.update", securityEvent = true, securityLevel = "high"),
     EmployeesRead("evaka.employees.read"),
```

Comment:
Do you think the read events should be classified as security events? My initial thought is: yes but `securityLevel = "low"` assuming nothing major is exposed with those events.

Reply:
Hmm, good point. Originally for some reason I thought not but now comparing to Person I will do that.